### PR TITLE
Signup1

### PR DESCRIPTION
**Pull request by junydania**
## What was done: 
      - Created a cucumber feature test to check the various input fields on the sign up page form

## What we learnt:
      - How to check the presence of input field and CSS selectors.
      - reinforcement of github forking and branching

### CODE REVIEW

**Review by diraulo — COMMENTED** [review]
Are you guys done with this or still busy working on it?

I don't see any scenario about the user actually filling up their details and getting an account. All you doing here is asserting that the fields are present on the page...

**Review by diraulo — COMMENTED** [review]
This looks fine to me now. Although the sad path for this scenario is missing. What happens when the user enters incorrect information and submits the form? or doesn't fill in any details at all?

File: features/sign_up.feature [hunk 1]
```diff
@@ -0,0 +1,57 @@
+Feature: Sign up Feature
+  As a user,
+  To sign up for an account,
+  I need a form to fill my information
+
+Background:
+  Given I visit the "Home" page
+  And I click on "Sign up" link
+  And I fill in field "user_name" with "John"
```
Comment (tochman) [inline features/sign_up.feature]: Didi you try with:
```
And I fill in field "Name" with "John"
```
More readible

**Review by tochman — COMMENTED** [review]
Just a quick comment

File: features/sign_up.feature (hunk 1 above)
Comment (junydania) [reply features/sign_up.feature]: This has been refactored as advised and the test passed.

**Review by AmberWilkie — APPROVED** [review]

**Pull request merged**